common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/NativeAuthRequestProvider.kt

```diff
@@ -224,32 +222,11 @@ class NativeAuthRequestProvider(private val config: NativeAuthOAuth2Configuratio
     //endregion
 
     //region /signup/start
-    internal fun createSignUpStartRequest(
-        commandParameters: SignUpStartCommandParameters
-    ): SignUpStartRequest {
-        LogSession.logMethodCall(TAG, "${TAG}.createSignUpStartRequest")
-
-        return SignUpStartRequest.create(
-            username = commandParameters.username,
-            attributes = commandParameters.userAttributes,
-            challengeType = config.challengeType,
-            clientId = config.clientId,
-            requestUrl = signUpStartEndpoint,
-            headers = getRequestHeaders()
-        )
-    }
-
     internal fun createSignUpUsingPasswordStartRequest(
```

Comment:
Let's rename this to remove the reference to password

Reply:
Done